contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortBuildAndPushJob.java

```diff
@@ -719,7 +722,7 @@ public String runBuildStore(Props props, String url) throws Exception {
 
     public void runPushStore(Props props, String url, String dataDir) throws Exception {
         // For backwards compatibility http timeout = admin timeout
-        int httpTimeoutMs = 1000 * props.getInt(PUSH_HTTP_TIMEOUT_SECONDS, 24 * 60 * 60);
+        int httpTimeoutMs = 1000 * props.getInt(PUSH_HTTP_TIMEOUT_SECONDS, 60);
```

Comment:
Fixed it, thanks for the great catch. I should have been careful while modifying it.